.circleci/config.yml

```diff
@@ -58,6 +70,10 @@ jobs:
           working_directory: content
           command: ./scripts/upload.sh
 
+      - slack/status:
+          success_message: ":terraformda: Successfully deployed to terraform.io."
+          failure_message: ":boom: Failed to deploy terraform.io! Urgent attention needed! <!here>"
```

Comment:
`<!here>` is API-ese for `@here`. https://api.slack.com/reference/surfaces/formatting#special-mentions